README.md

```diff
@@ -32,6 +32,7 @@ Database connection parameters and initialisation.
 - `omero_server_dbpassword`: Database password
 - `omero_server_rootpassword`: OMERO root password, default `omero`.
   This is only used when initialising a new database.
+- `omero_server_database_manage`: If `True` initialise or upgrade the OMERO database as required
```

Comment:
Also worth mentioning that the default is `True` (and maybe move to the top of the section)